src/Mailbox.cc

```diff
@@ -341,7 +360,8 @@ namespace kickcat
         : AbstractMessage(mailbox.recv_size)
         , mailbox_{mailbox}
     {
-
+        // Is this needed?
```

Comment:
To check before merge

Reply:
Tested successfully on an Elmo Gold (moving a motor and throwing CoE emergency message).